new-frontiers-14/frontier-station-14
# Adds ice sink

## About the PR



This PR adds a new sink type, an ice box, which slowly refills with ice for use in cooking environments. 

## Why / Balance


Ice is a somewhat finite resource at present, given the challenges in making it. As it has virtually no use that I could discern outside of limited culinary uses (time for a smoothie and ice cream PR) adding a mappable ice sink for bars and restaurants seemed like a good solution. 

## How to test


Place sink.
Take ice if full. Ensure it refills with ice and not, you know, coffee.
Check orientation sprites. Check drain sprites in all orientations. Try to not get a brain freeze.

## Media


![ss+(2024-12-30+at+11 16 33)](https://github.com/user-attachments/assets/4bbaafed-1482-4d34-bad8-ae6cd9dc09b7)

## Requirements



## Breaking changes


**Changelog**


🆑 
- add: mappable ice box for bars and restaurants!
